webpack/components/AnsibleHostDetail/components/AnsibleVariableOverrides/AnsibleVariableOverridesHelper.js

```diff
@@ -0,0 +1,8 @@
+export const reorderVariables = variables =>
```

Comment:
The naming here is confusing IMHO, it took me a while to understand what this function actually does. 
We aren't reordering the variables, we are flattening the response, and the argument isn't the variables, it's the roles.
How about something like:
```suggestion
export const extractVariables = roles =>
```